Doc/whatsnew/3.7.rst

```diff
@@ -409,6 +409,11 @@ therefore included in source distributions.
 and ``platforms`` fields are not specified as a list or a string.
 (Contributed by Berker Peksag in :issue:`19610`.)
 
+The ``upload`` command now longer tries to change CR end-of-line characters
+to CRLF.  This fixes a corruption issue with sdists that ended with a byte
+equivalent to CR.
+(Contributed by Bo Bayles in :issue:`32304`.)
```

Comment:
@bbayles I used the name found in your github profile, is that ok?

Reply:
Yes, that's fine. Thanks!